### PR TITLE
Blog xlsx direct importer

### PR DESCRIPTION
**Pull request by agl29**
## What changes were proposed in this pull request?

- blog post on supporting xlsx file in the direct importer

### CODE REVIEW

**Comment by agl29** [comment]
test1
this is direct_importer_xlsx.gif in blog.

If any correction is needed in this also let me know.

**Review by Harshg999 — APPROVED** [review]
LGTM!

File: docs/gethue/content/en/posts/2021-11-15-create-sql-tables-from-excel-files.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+ampforwp-amp-on-off:
+  - default
+categories:
+  - Version 4.11
+  - Development
+  - Query
+
+---
+
+If you are a Hue user then you might be familiar with the Hue [Direct Importer](/blog/2021-05-26-improved-hue-importer-select-a-file-choose-a-dialect-create-a-table/). It lets you create tables from a local CSV file. Now, you will be able to create SQL table from excel(.xlsx) file also.
```
Comment (pychheda) [inline docs/gethue/content/en/posts/2021-11-15-create-sql-tables-from-excel-files.md:48]: Please replace "Now, you will be able to create SQL table from excel(.xlsx) file also." with "Now you can also create SQL tables by importing Excel (.xlsx) files into Hue."

**Review by pychheda — APPROVED** [review]
Just a minor edit. The rest looks good.

File: docs/gethue/content/en/posts/2021-11-15-create-sql-tables-from-excel-files.md (hunk 1 above)
Comment (agl29) [reply docs/gethue/content/en/posts/2021-11-15-create-sql-tables-from-excel-files.md]: Done, 
Thanks

**Pull request merged**